File: Códigos/Ouve Fácil-SRC_01/class/cidade.php
<?php

class cidade{
    private $codCidade;
    private $nome;

    public function setCodCidade($codCidade){
        $this->codCidade = $codCidade;
    }
    public function getCodCidade(){
        return $this->codCidade;
    }
    public function setNome($nome){
        $this->nome = $nome;
    }
    public function getNome(){
        return $this->nome;
    }
}
?>

File: Códigos/OuveFacil/OuveFacil/app/src/main/java/domain/model/Cidade.java
package domain.model;

import java.util.ArrayList;

public class Cidade {

    private int codCidade;
    private String nome;
    private ArrayList<UF> uf = new ArrayList<>();

    public int getCodCidade() {
        return codCidade;
    }

    public void setCodCidade(int codCidade) {
        this.codCidade = codCidade;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public ArrayList<UF> getUf() {
        return uf;
    }

    public void setUf(ArrayList<UF> uf) {
        this.uf = uf;
    }

    @Override
    public String toString() {
        return this.nome;
    }
}
